_posts/2024-11-13-adbc-15-release.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: "Apache Arrow ADBC 15 (Libraries) Release"
+date: "2024-11-13 00:00:00"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+The Apache Arrow team is pleased to announce the 15 release of
+the Apache Arrow ADBC libraries. This covers includes [**31
+resolved issues**][1] from [**13 distinct contributors**][2].
```

Comment:
This language has been used in previous release blog posts but I just now noticed it's slightly awkward.

```suggestion
The Apache Arrow team is pleased to announce the version 15 release 
of the Apache Arrow ADBC libraries. This release includes [**31
resolved issues**][1] from [**13 distinct contributors**][2].
```

Reply:
I'll fix up the template